### ISSUE TITLE
uLauncher opens in primary desktop instead of active desktop in i3 after boot

### ISSUE THREAD

**Issue by kalenpw** [Ulauncher/Ulauncher#563]
### Steps to reproduce
1. In i3 config add `exec_always ulauncher --hide-window`
2. Boot into i3wm
3. Change to a desktop other than default one
4. Use your designated hotkey to open ulauncher
5. Observe that ulauncher is opened on the default desktop not the one you switched to

### Current behavior
After booting ulauncher will open in my first desktop instead of the currently active one. Eventually it will start launching in the correct one, and I haven't been able to narrow down what fixes it.

### Expected behavior
ulauncher always opens in the currently active window

### Environment
**Ulauncher version**: v5.7.5
**Linux distro**: Xubuntu 18.04
**Desktop environment**: i3

### Logs

https://gist.github.com/kalenpw/a78925eb1a52c00b056c984cc3b35e98

**Comment by mahmadmujtaba** [comment]
does it boot into primary desktop rather than expected desktop? i think the issue belongs to selection in Ulauncher options to open it where cursor is and upon boot cursor is at default at primary monitor so ulauncher opens there and afterwards it opens into where the settlings lead to.

**Comment by kalenpw** [comment]
Yes, I have my settings to monitor with a mouse pointer, I tried switching to a different workspace, wiggling my mouse in case the pointer didn't update until it was moved and then launching uLauncher and experienced that same behavior.

**Comment by friday** [comment]
This should be fixed in e0bac2c to be released very soon.
